docs/pbc/all/payment-service-provider/202212.0/payments-feature-overview.md

```diff
@@ -0,0 +1,92 @@
+* [BS Payone](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/bs-payone/bs-payone.html)
+* [Computop](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/computop/computop.html)
+* [CrefoPay](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/crefopay/crefopay.html)
+* [Heidelpay](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/heidelpay/heidelpay.html)
+* [Klarna](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/klarna/klarna.html)
+* [Payolution](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/payolution/payolution.html)
+* [Powerpay](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/powerpay.html)
+* [ratenkauf by easyCredit](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/ratenkauf-by-easycredit/ratenkauf-by-easycredit.html)
+* [RatePay](/docs/pbc/all/payment-service-provider/{{page.version}}/third-party-integrations/ratepay/ratepay.html)
+
+## Dummy payment
+
+By default, Spryker provides the [DummyPayment](https://github.com/spryker/dummy-payment) module, which has credit card and invoice payments implemented. You can use these implemented payment methods or refer to the DummyPayment module when implementing additional payment methods in your project.
+For details about implementing a new payment method, see [how to implement the Direct Debit payment method](/docs/scos/dev/back-end-development/data-manipulation/payment-methods/direct-debit-example-implementation/implementing-direct-debit-payment.html). Based on the examples in these documents, you can implement other payment methods for your projects.
+
+## Payment methods in the Back Office
+
+In the Back Office, you can view all payment methods available in the shop application and make a payment method active (visible) or inactive (invisible) in the **Payment** step of the checkout process. In addition, you can define stores in which a payment method is displayed. If changed, the payment methods are updated in the checkout as well.
+
+{% info_block warningBox "Note" %}
+
+Before managing payment methods in the Back Office, you need to create them by [importing payment methods data using a .CSV file](/docs/pbc/all/payment-service-provider/{{page.version}}/import-data/file-details-payment-method.csv.html).
+
+{% endinfo_block %}
+
+![List of payment methods](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Payment/Payment+Methods+Overview/payment-methods-list.png)
+
+To learn more on how to make a payment method available during the checkout and assign it to a different store, see [Edit payment methods](/docs/pbc/all/payment-service-provider/{{page.version}}/manage-in-the-back-office/edit-payment-methods.html).
+
+<!-- Managing Payment Methods in the Back Office
```

Comment:
We shouldn't keep commented content. I suggest removing it.
```suggestion
<!-- Managing Payment Methods in the Back Office
```

Reply:
Out of scope of this PR. I didn't write this document, just moving it around